source/plugin/data/serialization.rst

```diff
@@ -123,3 +125,56 @@ from persistent files using the :doc:`Configurate Library <../configuration/inde
         final DataView container = translator.translate(node);
         return deserializeHealth(container);
     }
+
+
+DataFormat
+==========
+An alternative to DataTranslators is to use :javadoc:DataFormat, which allows you to store a ``DataContainer`` as a hocon, json or NBT file. You can also retrieve data using ``DataFormat``.
+
+This is very useful if you're for example using a database to store information, as you can then serialize any ``DataContainer`` to json format.
+
+For this example we will use :javadoc:`DataFormats#JSON` to translate a player to json and back! 
+
+**Imports for code examples**
+    import org.spongepowered.api.data.DataContainer;
+    import org.spongepowered.api.data.persistence.DataFormat;
+    import org.spongepowered.api.data.persistence.DataFormats;
+    import org.spongepowered.api.entity.living.player.Player;
+
+    import java.io.ByteArrayOutputStream;
+    import java.io.IOException;
+    import java.io.OutputStream;
+    import java.io.InputStream;
+    import java.util.Optional;
+
+**Code Example: Serializing a Player to json format**
+    
+.. code-block:: java
+
+    public OutputStream toJsonFormat(Player player){
```

Comment:
It might be better to use a specific example, like writing a `ItemStack` to a file and loading it back from it. You use an arbitrary `ByteArrayOutputStream` here which is returned as `OutputStream` so the return value of this method is essentially useless without additional casting. Additionally, how often do you actually serialize an entire `Player` on your own, and how would you load it back? The example below does only show how to read it back into a `DataContainer`.